### ISSUE TITLE
Gathering money from alt business

### ISSUE THREAD

**Issue by LL782** [LL782/LL782#13]
## Why

1. We need money for growing the new business
2. It's a good way to touch base with old business
3. It should reduce distractings from the old business

## What

- [ ] Gather up hosting money from LesFilmsDuGeste
- [ ] Gather up hosting money from Tom Knutson
- [ ] Gather up hosting money from Legend

## Notes

1. Arranging regular payments for each of these ongoing is another story
2. Don't worry about other outstanding payments (Ross, Simon P, etc.) it's just easier
3. Make this quick and easy

**Comment by LL782** [comment]
Closing as a duplicate of https://github.com/LL782/LL782/issues/24

**Comment by LL782** [comment]
Reopening as plans keep changing. The latest news (after #52) is that we'd rather #24 than close it down.

We still use self hosted Wordpress installs and don't want to change that necessarily.

**Comment by LL782** [comment]
This has been hanging over me and having a task here on the board isn't going to make it happen any quicker. If it gets done it's a bonus but it's not needed on the critical path
